src/Workspaces/Core/Portable/CodeActions/CodeActionPriority.cs

```diff
@@ -2,25 +2,29 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using Microsoft.CodeAnalysis.Diagnostics;
+
 namespace Microsoft.CodeAnalysis.CodeActions
 {
+#pragma warning disable CA1200 // Avoid using cref tags with a prefix
```

Comment:
Might be worth adding a comment that you're using this because you're referencing a higher layer.

Reply:
Will do